packages/clima_data/lib/data_sources/forecast_remote_data_source.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:convert';
+
+import 'package:clima_core/failure.dart';
+import 'package:clima_data/models/forcasts_model.dart';
+import 'package:clima_domain/entities/city.dart';
+import 'package:dartz/dartz.dart';
+import 'package:http/http.dart' as http;
+import 'package:riverpod/riverpod.dart';
+
+const String apiKey = '4bef3adf2fcb90307c2bf5feac75a2ba';
```

Comment:
This should probably be put in a separate `constants.dart` file in `clima_data` or something.